test/integration/rules/nested-interactive/nested-interactive.html

```diff
@@ -3,12 +3,15 @@
 <div role="tab" id="pass3">pass</div>
 <div role="checkbox" id="pass4">pass</div>
 <div role="radio" id="pass5"><span>pass</span></div>
+<div role="button" id="pass6"><button aria-hidden="true" tabindex="-1">pass</button></div>
```

Comment:
There's already an `id="pass6"` on line 10.

```suggestion
<div role="button" id="pass7"><button aria-hidden="true" tabindex="-1">pass</button></div>
```